wgsl/index.bs

```diff
@@ -638,6 +643,9 @@ A processing error may occur during three phases in the shader lifecycle:
     is an error detectable at [=pipeline creation=] time.
     Detection relies on the WGSL module source text
     and other information available to the particular pipeline creation API method.
+    These errors are only triggered for code present in the
```

Comment:
Also need to include statically accessed workgroup variables.
It feels wrong to include workgroup variables in the _interface_.

It turns out we already define the [shader](https://gpuweb.github.io/gpuweb/wgsl/#shader) as having exactly the right stuff: the supporting code, the statically accessed variables (of all kinds) and constants, and types.

Otherwise I would have suggested addin the concept of the _support_ of an entry point, but "shader" is exactly that.